### ISSUE TITLE
Help needed!

### ISSUE THREAD

**Issue by grafanaKibana** [floookay/pluckey#6]
Hi! I will say right away that I am a complete beginner, but I always wanted to do something like this. I very liked your layout, but for me will be great to add 2 more buttons on each side. It would be great if you could help me with this :) 
![image](https://user-images.githubusercontent.com/45916288/150762319-e8c998de-b8fe-4310-a7d9-bcc7f9c87445.png)

**Comment by floookay** [comment]
Hi,

sure, I can guide you on what needs to be done:
First you’d need to add the two extra keys to the matrix of the pcb. You can do that with kicad and eeschema. You probably have to add another column though because I mapped a thumb key there iirc in order to reduce the amount of needed column pins.
After you’ve done that you can edit the pcb itself with pcbeditor by importing the new schema. Set the grid to 19.05/8mm and set the offset to the position of one of the outermost keys.
Now you can move the new keys to the right positions and update the traces.
You can run the built in check once your done to check if you missed something.

For the case you’d be probably better of creating something new entirely.
I don’t think the current cases (except maybe skeleton) are editable in a way to add those keys efficiently.
For a new case I would open the mono case file, delete all parts except the plate, update said plate with the new keys and build from there.

**Comment by aldecode** [comment]
> Hi, sure, I can guide you on what needs to be done: First you’d need to add the two extra keys to the matrix of the pcb. You can do that with kicad and eeschema. You probably have to add another column though because I mapped a thumb key there iirc in order to reduce the amount of needed column pins. After you’ve done that you can edit the pcb itself with pcbeditor by importing the new schema. Set the grid to 19.05/8mm and set the offset to the position of one of the outermost keys. Now you can move the new keys to the right positions and update the traces. You can run the built in check once your done to check if you missed something. For the case you’d be probably better of creating something new entirely. I don’t think the current cases (except maybe skeleton) are editable in a way to add those keys efficiently. For a new case I would open the mono case file, delete all parts except the plate, update said plate with the new keys and build from there.

Sounds super complicated to me :) Apparently I'll have to use the option as it is. But thanks for the mini-guide, I think it might be useful to me in the future

**Comment by floookay** [comment]
Yeah, it’s a quite a bit of work in this case. But if you have any specific questions feel free to email me :)

Also, if you’re starting with pcb design I’d recommend this guide: https://github.com/ruiqimao/keyboard-pcb-guide
It helped me greatly when I started

**Comment by aldecode** [comment]
@floookay many thanks!
